src/RenderState.hpp

```diff
@@ -10,10 +10,10 @@ struct Gosu::RenderState
     std::shared_ptr<Texture> texture;
     const Transform* transform;
     ClipRect clip_rect;
-    AlphaMode mode;
+    BlendMode mode;
     
     RenderState()
-    : transform(0), mode(AM_DEFAULT)
+    : transform(0), mode(BM_DEFAULT)
```

Comment:
Can you please update this to `transform(nullptr)` while you're touching this line? Or even better, you can initialize the members inline using `=`